smarts/core/waymo_map.py

```diff
@@ -112,6 +116,7 @@ def __init__(self, map_spec: MapSpec, waymo_scenario):
         self._default_lane_width = WaymoMap.DEFAULT_LANE_WIDTH
         self._lane_rtree = None
         self._no_composites = False  # for debugging purposes
+        self.has_overpasses = False
```

Comment:
I don't think we want to allow class users to be able to set this member.  This, I would suggest using `self._has_overpasses` and then exposing that as a property.  Also, I think this property is worth adding to the `RoadMap` interface too (with a default implementation there that just returns `False` so the other maps don't have to implement it themselves).